src/app/child-dev-project/children/children.service.spec.ts

```diff
@@ -72,30 +72,6 @@ describe("ChildrenService", () => {
 
   // TODO: test getAttendances
 
-  it("should find latest ChildSchoolRelation of a child", async () => {
```

Comment:
why are these tests not relevant anymore? Is there any other test case covering to check what will happen if 0, 1 or more relations are found and whether the correct one is picked for the child?